src/platform-integration/android/install-android/install-android-from-web-on-linux.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Add Android devtools for Flutter from Web on Linux start
```

Comment:
I am exceedingly confused by what this means. First of all, "devtools" is not a word. Flutter and Dart and Flutter (or Dart) DevTools. Chrome has Chrome DevTools. There, so far as I know, aren't any "web DevTools". I don't even understand this title.